### PR TITLE
AG fixes, rename metrics collections, restrcuture installation procedure

### PR DESCRIPTION
**Pull request by yhontyk**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Restructures the procedure, a few improvements and fixes, renames metrics collections to traces collections.

### CODE REVIEW

File: src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-operator.adoc [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
 
-Navigate to your OpenShift cluster's console, and select *Operators*.  Select *OperatorHub* and type "image puller" into the "Filter by keyword.." search bar.  Click the *Kubernetes Image Puller Operator*, click *Continue* and click *Install*.  At the *Installation Mode* selection, choose *A specific {orch-namespace} on the cluster*, and use the drop-down to find the {orch-namespace} you created to install the image puller.  Click *Subscribe*.  
+. Wait for the Image Puller Operator to install. Click the btn:[*KubernetesImagePuller*] → btn:[*Create instance*]. 
 
-Wait for the Kubernetes Image Puller Operator to install, and click the installation.  Click the *KubernetesImagePuller* tab, and then click *Create instance*.  You will be taken to a screen with a YAML editor with a `KubernetesImagePuller` Custom Resource.  Make any modifications to the Custom resource and click *Create*.
+. You will be redirected to a screen with a YAML editor with a `KubernetesImagePuller` Custom Resource. Make any modifications to the Custom resource and click btn:[*Create*].
```
Comment (themr0c) [inline src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-operator.adoc]: Passive voice

File: src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-operator.adoc [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
+
 [id="deploying-image-puller-using-operator_{context}"]
 = Deploying Image Puller using the Operator
 
 The recommended way to deploy the Image Puller is through the link:https://github.com/che-incubator/kubernetes-image-puller-operator[Operator].
 
 == Installing the Image Puller on OpenShift using OperatorHub
 
-First, create a {orch-namespace} in your cluster to host the image puller.  Our example will use the {orch-namespace} "image-puller".
+First, create a {orch-namespace} in your cluster to host the image puller. This document uses the {orch-namespace} "image-puller" as an example.
```
Comment (themr0c) [inline src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-operator.adoc]: This should be either in a .Prerequisites section or a first step.

File: src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-operator.adoc [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
+
+.Procedure
+. Navigate to your OpenShift cluster console, navigate to btn:[*Operators*] → btn:[*OperatorHub*].  
+
+. Use the btn:[*Filter by keyword*] box to search for `Kubernetes Image Puller Operator`. Click the btn:[Kubernetes Image Puller Operator].
+
+. Read the descriotion of the Operator. Click btn:[*Continue*] → btn:[*Install*].
```
Comment (themr0c) [inline src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-operator.adoc]:
```suggestion
. Read the description of the Operator. Click btn:[*Continue*] → btn:[*Install*].
```

File: src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-operator.adoc (hunk 1 above)
Comment (yhontyk) [reply src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-operator.adoc]: @themr0c made some gymnastics with this one. PTAL

**Review by themr0c — APPROVED** [review]

**Pull request merged**